test/ServerRequestFactoryTestCase.php

```diff
@@ -50,11 +50,11 @@ public function dataMethods()
         ];
     }
 
-    public function dataServer()
+    public static function dataServer()
     {
         $data = [];
 
-        foreach ($this->dataMethods() as $methodData) {
+        foreach (self::dataMethods() as $methodData) {
```

Comment:
This should be `static::` to mimic the old behaviour of being able to override dataMethods in the parent.

Reply:
Good catch!